.goreleaser.yml

```diff
@@ -36,10 +36,10 @@ builds:
     ignore:
       - goos: windows
         goarch: arm64
-
-  - id: cosign
-    dir: plugins/verifier/cosign
-    binary: cosign
+  
+  - id: vulnerabilityreport
```

Comment:
probably also need to add the sbom verifier here too, but can be different PR.

Reply:
I think we're already covered? I found a pre-existing SBOM publishing step on Line 24.